src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -20,67 +22,108 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Intake extends SubsystemBase {
-  private final double intakePivotMotorGearRatio = 100.0;
+
+  // local constants
+
+  // IDs
   private final int intakeMotorID = 10;
   private final int intakePivotID = 11;
   private final int intakePivotEncoderID = 9;
+
+  // positions
+  // NOTE: these positions are also used in robotcontainer.
+  public final double intakeSafePosition = 105.73;
+  public final double intakeGroundPosition = 0;
+  public final double intakeLoadPosition = 0;
+
+  // shooter PID
   private final double intakePValue = 0.2;
   private final double intakeIValue = 0.0;
   private final double intakeDValue = 0.0;
+
+  // feed forward PID
   private final double intakeSValue = 0.0;
   private final double intakeGValue = 0.0;
   private final double intakeVValue = 0.0;
+
+  private final double intakePivotMotorGearRatio = 100.0;
+
   private final double magnetOffSet = 0.0;
 
+
+  // local variables
+  private double m_setPoint;
+  private double m_IntakePiviotVoltage = 0.0;
+
+  // WPILib class objects
   private CANSparkMax intakeMotor;
   private CANSparkMax m_IntakePiviot;
   private CANcoder intakePivotEncoder;
   private RelativeEncoder intakePivotMotorEncoder;
   private PIDController pid;
   private ArmFeedforward intakePivotFeedforward;
-  private double m_setPoint;
-  private double m_IntakePiviotVoltage = 0.0;
 
 
-  /** Creates a new Intake. */
+  // constructor
   public Intake() {
+
+    // instantiate objects
     intakeMotor = new CANSparkMax(intakeMotorID, MotorType.kBrushless);
     m_IntakePiviot = new CANSparkMax(intakePivotID, MotorType.kBrushless);
     intakePivotEncoder = new CANcoder(intakePivotEncoderID);
-    CANcoderConfigurator configuator = intakePivotEncoder.getConfigurator();
 
+    // configure cancoder
+    CANcoderConfigurator configuator = intakePivotEncoder.getConfigurator();
     configuator.apply(
       new MagnetSensorConfigs()
         .withAbsoluteSensorRange(AbsoluteSensorRangeValue.Signed_PlusMinusHalf)
         .withMagnetOffset(magnetOffSet)
     );
 
+    // set integrated encoder to position from cancoder
+
+    // TODO: Go over this part with Dylan and student that wrote this. Can we simplify this?
```

Comment:
Making this a comment so we remember to do this at some point (probably tomorrow)